doc/api/n-api.md

```diff
@@ -3155,6 +3155,21 @@ support it:
 * If the function is not available, provide an alternate implementation
   that does not use the function.
 
+### napi_adjust_external_memory
```

Comment:
I think you would need a "Garbage Collection" heading here as well, so as to not categorize it under "Version Management".